config/ie/options

```diff
@@ -316,12 +317,22 @@ ALEMBIC_LIB_SUFFIX = "-" + alembicVersion
 
 # find USD:
 
+USD_INCLUDE_PATH = None
+USD_LIB_PATH = None
+
 if usdVersion:
-    USD_INCLUDE_PATH = os.path.join( "/software/apps/usdgaffer/", usdVersion, platform, "include" )
-    USD_LIB_PATH = os.path.join( "/software/apps/usdgaffer/", usdVersion, platform, "lib" )
-else:
-    USD_INCLUDE_PATH = None
-    USD_LIB_PATH = None
+
+	usdReg = IEEnv.registry["apps"]["usd"][usdVersion][platform]
+
+	if usdReg:
+
+		if targetApp:
+			USD_INCLUDE_PATH = os.path.join( usdReg["location"], targetApp, compatibilityVersion,"include" )
+			USD_LIB_PATH = os.path.join( usdReg["location"], targetApp, compatibilityVersion, "lib" )
+		else:
+			USD_INCLUDE_PATH = os.path.join( usdReg["location"], "cortex", "10", "include" )
```

Comment:
Can we use `$IECORE_MAJOR_VERSION` instead of hard-coding "10"?
If that works, I also think it would make sense to update the hard-coded "10" in line 66.